install_config/persistent_storage/persistent_storage_flex_volume.adoc

```diff
@@ -75,43 +75,41 @@ spec:
-claims] or from pods. This name may be different than the name of the volume
-on back-end storage.
+claims] or from pods. This name can be different from the name of the volume on
+back-end storage.
 <2> The amount of storage allocated to this volume.
 <3> Name of the driver. This field is mandatory.
 <4> Optional file system that is present on the volume.
-<5> Optional reference to a secret. Keys and values from this secret will be
-provided to the Flex Volume driver on invocation.
+<5> Optional reference to a secret. Keys and values from this secret are
+provided to the FlexVolume driver on invocation.
 <6> Optional read-only flag.
-<7> Additional options for the Flex Volume driver. This is a free-form
-dictionary of parameters provided to the Flex Volume driver. Typically, it
-includes details like the name of the volume that this `PersistentVolume`
-represents and addresses of the storage server(s).
-====
+<7> Additional options for the FlexVolume driver. This is a free-form dictionary
+of parameters provided to the FlexVolume driver. Typically, it includes details
+like the name of the volume that this `PersistentVolume` represents and
+addresses of the storage server(s).
+
 
 [[flex-volume-drivers]]
-=== Flex Volume drivers
+== FlexVolume Drivers
 
-As noted above, a Flex Volume driver is an executable, typically a shell script,
-that resides in a well-defined directory on all machines in the cluster, both
-masters and nodes. {product-title} calls it whenever it needs to attach, detach,
-mount, or unmount a volume represented by a `PersistentVolume` with `flexVolume`
-as source.
+A FlexVolume driver is an executable file that resides in a well-defined
+directory on all machines in the cluster, both masters and nodes.
+{product-title} calls it whenever it needs to attach, detach, mount, or unmount
+a volume represented by a `PersistentVolume` with `flexVolume` as source.
 
-The first command-line argument of the driver is always an operation name.
-Other parameters are specific to each operation. Most of the operations expect
-JSON string as a parameter. This parameter is literally a JSON string,
-not name of a file with JSON!
+The first command-line argument of the driver is always an operation name. Other
+parameters are specific to each operation. Most of the operations takes a
+JSON(JavaScript Object Notation) string as a parameter. This parameter is a
+complete JSON string, and not the name of a file with the JSON data.
 
 It contains:
 
-* All `flexVolume.options`.
-* Some options from `flexVolume` prefixed by `kubernetes.io/`, such as `fsType`
-  and `readwrite`.
-* Content of the referenced secret (if specified) prefixed by `kubernetes.io/secret/`.
+* All `flexVolume.options`. Some options from `flexVolume` prefixed by
+* `kubernetes.io/`, such as `fsType` and `readwrite`. Content of the referenced
+* secret (if specified) prefixed by `kubernetes.io/secret/`.
```

Comment:
I think there's been a formatting mistake above. Should the points be on different lines?